examples/_dev/src/components/Asset.tsx

```diff
@@ -2,58 +2,73 @@ import { useCreateAsset, useUpdateAsset } from '@livepeer/react';
 import { useState } from 'react';
 
 export const Asset = () => {
-  const [video, setVideo] = useState<File | undefined>();
-  const { mutate: createAsset, data: asset, status } = useCreateAsset();
-  // const { data: asset } = useAsset({
-  //   assetId: createdAsset?.id,
-  //   refetchInterval: 10000,
-  // });
+  const [videos, setVideos] = useState<File[]>([]);
+
+  const { mutate: createAssets, data: assets, status } = useCreateAsset();
+
   const { mutate: updateAsset, status: updateStatus, error } = useUpdateAsset();
 
+  const handleCreateAsset = async () => {
+    const files = videos.map((video) => ({
+      file: video,
+      name: video.name,
+    }));
+
+    createAssets(files);
+  };
+
   return (
     <div>
       <input
         type="file"
+        multiple
         accept="video/*"
-        onChange={(e) => setVideo(e?.target?.files?.[0])}
-      />
-      <button
-        disabled={!video || status === 'loading'}
-        onClick={() => {
-          if (video) {
-            createAsset({
-              name: video.name,
-              file: video,
-            });
+        onChange={(e) => {
+          if (e.target.files) {
+            setVideos(Array.from(e.target.files));
```

Comment:
ES6-ey way of doing this would be

```suggestion
            setVideos([...e.target.files]);
```